### PR TITLE
Add potential-robot to Builder Tools

### PR DESCRIPTION
**Pull request by aleeusgr**
👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.

1. Please complete a Checklist
2. Fill in all sections of the template
3. Click "Create pull request"

## Checklist

 <-- Please fill the boxes with [x] before submitting a pull request --> 

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have read the [Builder Tool Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/builder-tools.js)
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Builder Tool addition

<-- Provide information for every bullet in the list below. The tags you select must match the tags in your changes to the builder-tools.js -->

* Title: *Project name*
* Description: *Describe your project* 
* Website:   <link_to_project>
* Getstarted: <link_to_getstarted_page>
* Tags:
  * `chainindex`
  * `cli`
  * `golang`
  * `IDE`
  * `java`
  * `javascript`
  * `marlowe`
  * `net`
  * `nft`
  * `operatortool`
  * `oracle`
  * `plutus`
  * `python`
  * `rust`
  * `purescript`
  * `websocket`
  * `http`
  * `json`
  * `haskell`
  * `typescript`
  * `wallet`
  * `serialization`
  * `sdk`
  * `redis`
  * `sql`
  * `lowlevel`
  * `testing`
  * `hosted`

### CODE REVIEW

File: src/data/builder-tools.js [hunk 1]
```diff
@@ -1034,6 +1034,13 @@ export const Showcases = [
     getstarted: "https://docs.gomaestro.org/",
     tags: ["http", "hosted"]
   },
+  {
+    title: "potential-robot",
+    description: "A JavaScript API for test-driven development with Helios",
```
Comment (rphair) [inline src/data/builder-tools.js:1039]:
```suggestion
    description: "A JavaScript API for test-driven development with Helios.",
```

**Review by rphair — APPROVED** [review]
please update, for consistency with other project `description` fields:

**Comment by aleeusgr** [comment]
Many thanks, English isn't my native tongue.

**Comment by rphair** [comment]
good, then please note that we are punctuating all of these `description` fields wrong.  In English, a period is only supposed to end a complete sentence... but as you can see all these descriptions are noun phrases.  I apologise for all of us setting such a bad example, although the way it's written now is what people expect to see in the online world. 🤓

File: src/data/builder-tools.js [hunk 2]
```diff
@@ -1034,6 +1034,13 @@ export const Showcases = [
     getstarted: "https://docs.gomaestro.org/",
     tags: ["http", "hosted"]
   },
+  {
+    title: "potential-robot",
+    description: "A JavaScript API for test-driven development with Helios.",
+    preview: require("./builder-tools/potential-robot.png"),
+    website: "https://github.com/aleeusgr/potential-robot",
+    tags: ["javascript", "typescript", "testing"]
```
Comment (rphair) [inline src/data/builder-tools.js:1042]:
```suggestion
    getstarted: null,
    tags: ["javascript", "typescript", "testing"]
```
oops, we both forgot this required field (the Portal will not build unless this is present for every entry)

**Comment by aleeusgr** [comment]
Thanks again.

Despite me complaining a lot about catalyst, Dev community is the best part of Cardano.

Thank you for you help🙇‍♂️

**Review by fill-the-fill — APPROVED** [review]

**Review by os11k — APPROVED** [review]

**Pull request merged**